WalletWasabi.Tests/UnitTests/Data/MempoolInfoWithHistogram.json

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
Can you share where this data comes from?

Reply:
```
$ bitcoin-cli getmempoolinfo true
```